diff --git a/node_modules/react-native-smooth-pincode-input/src/index.js b/node_modules/react-native-smooth-pincode-input/src/index.js
index 0a00f22..33b1ac8 100644
--- a/node_modules/react-native-smooth-pincode-input/src/index.js
+++ b/node_modules/react-native-smooth-pincode-input/src/index.js
@@ -6,7 +6,8 @@ import {
   TextInput,
   StyleSheet,
   I18nManager,
-  ViewPropTypes,
+  ViewStyle,
+  TextStyle,
 } from 'react-native';
 import * as Animatable from 'react-native-animatable';
 
@@ -286,14 +287,14 @@ SmoothPinCodeInput.propTypes = {
 
   restrictToNumbers: PropTypes.bool,
 
-  containerStyle: ViewPropTypes.style,
+  containerStyle: ViewStyle,
 
-  cellStyle: ViewPropTypes.style,
-  cellStyleFocused: ViewPropTypes.style,
-  cellStyleFilled: ViewPropTypes.style,
+  cellStyle: ViewStyle,
+  cellStyleFocused: ViewStyle,
+  cellStyleFilled: ViewStyle,
 
-  textStyle: Text.propTypes.style,
-  textStyleFocused: Text.propTypes.style,
+  textStyle: TextStyle,
+  textStyleFocused: TextStyle,
 
   animated: PropTypes.bool,
   animationFocused: PropTypes.oneOfType([
